Docs/developer_guide/script_repository/dicom.md

```diff
@@ -260,15 +263,15 @@ dicomRetrieve.callingAETitle = dicomQuery.callingAETitle
 dicomRetrieve.calledAETitle = dicomQuery.calledAETitle
 dicomRetrieve.host = dicomQuery.host
 dicomRetrieve.port = dicomQuery.port
-dicomRetrieve.setMoveDestinationAETitle("SLICER");
+dicomRetrieve.setMoveDestinationAETitle("SLICER")
 dicomRetrieve.setDatabase(slicer.dicomDatabase)
-for study in dicomQuery.studyInstanceUIDQueried:
-  print(f"ctkDICOMRetrieveTest2: Retrieving {study}")
+for study, series in dicomQuery.studyAndSeriesInstanceUIDQueried:
+  print(f"ctkDICOMRetrieveTest: Retrieving {study}")
   slicer.app.processEvents()
   if dicomQuery.preferCGET:
-    success = dicomRetrieve.getStudy(study)
+    success = dicomRetrieve.getSeries(series)

```

Comment:
@lassoan I think the getSeries(), moveSeries() needs both, the `study` key and the `series` key

Reply:
That's correct. Somehow I missed this when I copied the changes from my Python console. I've updated it now. Please test.